mne/decoding/transformer.py

```diff
@@ -12,42 +12,72 @@
 from .. import pick_types
 from ..filter import filter_data, _triage_filter_params
 from ..time_frequency.psd import _psd_multitaper
-from ..externals import six
-from ..utils import _check_type_picks
+from ..utils import _check_type_picks, requires_sklearn, check_version
+from ..io.pick import pick_info, _pick_data_channels, _picks_by_type
+from ..cov import _undo_scaling_array, _apply_scaling_array
 
 
-class Scaler(TransformerMixin):
-    """Standardize data across channels.
+@requires_sklearn
+class Scaler(TransformerMixin, BaseEstimator):
+    """Standardizes data across channels.
 
     Parameters
     ----------
+    scalings : dict, string, defaults to "standard"
+        if scalings is 'robust' sklearn.proprocessing.RobustScaler() is used.
+        robust requires sklearn version 0.17.
+        if scalings is 'standard' sklearn.preprocessing.StandardScaler()
+        is used.
+        if scalings is 'default', scales mag by 1e15
```

Comment:
@jasmainak I am not sure what other channel types should be scaled with. I used the value as used in the `cov` module [here](https://github.com/mne-tools/mne-python/blob/master/mne/cov.py#L1876).